x-pack/plugins/security_solution/public/common/components/markdown_editor/plugins/index.ts

```diff
@@ -27,14 +27,30 @@ export const {
 
 export const platinumOnlyPluginTokens = [insightMarkdownPlugin.insightPrefix];
 
-export const uiPlugins = ({ licenseIsPlatinum }: { licenseIsPlatinum: boolean }) => {
+export const uiPlugins = ({
+  licenseIsPlatinum,
+  upsellInvestigationGuide,
```

Comment:
Perhaps renaming this to `insightsUpsellingMessage` would be more appropriate, as the current name may be misleading as a _boolean_.

Reply:
I am confused by this comment. This PR is about the investigation guide and not `insights`. Why boolean? I like the suggestion of sufizing the variable name with `Message`.

Reply:
Oh, the feature is also called `insight`. 😕